docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -70,6 +76,15 @@ GROUP BY GROUPING SETS (grouping_set [ , ...])
     ((warehouse, product), (warehouse), (product), ())</code>.
     The N elements of a <code>CUBE</code> specification results in 2^N <code>GROUPING SETS</code>.
   </dd>
+  <dt><code><em>aggregate_name</em></code></dt>
+  <dd>
+    Specifies an aggregate function name (MIN, MAX, COUNT, SUM, AVG, etc.).
+  </dd>
+  <dt><code><em>FILTER</em></code></dt>
```

Comment:
not related to this PR but would be great to explain DISTINCT as well.

Reply:
ok, I'll add.